url/url.pxd

```diff
@@ -14,7 +14,7 @@ cdef extern from "url-cpp/include/psl.h" namespace "Url":
         @staticmethod
         PSL fromString(const string& str)
         string getTLD(const string& hostname) const
```

Comment:
`getTLD` should have an equivalent change as well. I've got a branch I was doing this same code on though, so I'll just push mine.

Reply:
Great. I've been unable to trigger the C++ exception from getTLD in a test case. I did add a test case for the getPLD call.